tests/test_leftovers.py

```diff
@@ -1,38 +1,39 @@
 from indexd.blueprint import get_record
 from tests.test_client import get_doc
 import pytest
-from indexd.index.errors import NoRecordFound as IndexNoRecordFound
-from indexd.dos.blueprint import get_dos_record
 import json
 
 
-def test_get_record_call_dist_get_record(client, user):
-    # test get_record correctly
+def test_get_record_error(client, user):
+    # test getting an existing ID
     data = get_doc()
     res = client.post("/index/", json=data, headers=user)
     assert res.status_code == 200
     rec = res.json
-    response = get_record(rec["did"])
-    record = response[0].json
-    assert record["urls"][0] == "s3://endpointurl/bucket/key"
-    assert record["size"] == 123
+    response = client.get("/index/" + rec["did"])
```

Comment:
good catch, I think it was covered very sneakily in one assert statement of the original tests. I wrote another more obvious test for it as well just in case, I can delete it though if need be.